### PR TITLE
Threlte refactor

### PR DESCRIPTION
**Pull request by yustarandomname**

### CODE REVIEW

**Comment by juliavdkris** [comment]
bruh
image

**Comment by yustarandomname** [comment]
> bruh image

image
😃

**Comment by juliavdkris** [comment]
But looks great! One thing: I suggest changing the `server.port` in the vite config from 5173 to 3000. When building for prod it actually does still seem to use 3000, so 5173 being the new default doesn't break anything fortunately, but juuust to be sure :)

**Comment by yustarandomname** [comment]
> But looks great! One thing: I suggest changing the `server.port` in the vite config from 5173 to 3000. When building for prod it actually does still seem to use 3000, so 5173 being the new default doesn't break anything fortunately, but juuust to be sure :)

oki is goed

**Comment by juliavdkris** [comment]
Mainly just because 5173 is a weird number that does not spark joy, but also because if the prod build uses a different port that would actually break stuff (since the Docker image expects it to run on port 3000)

**Review by juliavdkris — APPROVED** [review]

File: apps/web/package.json [hunk 1]
```diff
@@ -1,37 +1,42 @@
 {
   "name": "web",
-  "version": "0.0.0",
+  "version": "0.0.1",
+  "private": true,
   "scripts": {
-    "dev": "vite dev --port 3000",
+    "dev": "vite dev",
```
Comment (yustarandomname) [inline apps/web/package.json:6]:
```suggestion
    "dev": "vite dev --port 3000",
```

Comment (juliavdkris) [reply apps/web/package.json:6]: Ah I already set it in the vite config itself :)

Comment (juliavdkris) [reply apps/web/package.json:6]: Yesss!

https://vitejs.dev/config/server-options.html

**Comment by juliavdkris** [comment]
Why does GitHub keep detecting merge conflicts when there aren't any??

image

image

**Pull request merged**